recipes/microhapulator/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.1.3" %}
+{% set sha256 = "d613d1c4ec977feab1f3ec7a53835b6b2bde9d48af2036c2b910a4d1d943641a" %}
+
+package:
+  name: microhapulator
+  version: '{{version}}'
+
+source:
+  url: https://github.com/bioforensics/microhapulator/archive/{{ version }}.tar.gz
+  sha256: '{{sha256}}'
+
+build:
+  noarch: python
+  entry_points:
+    - mhpl8r = microhapulator.__main__:main
+  script: python -m pip install --no-deps --ignore-installed .
+  number: 0
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - happer >=0.1
+    - insilicoseq >=1.3.6
+    - microhapdb >=0.3
+    - numpy >=1.15.4
+    - python >=3
+    - pyfaidx >=0.5.5.2
+    - pysam >=0.15.2
```

Comment:
Want to pin that one to `>=0.15.2,<0.16.0a0`? I think they changed API before on patch level bumps.

Reply:
Good to know. I'll consider that for the next update.

Reply:
Addressed in https://github.com/bioconda/bioconda-recipes/pull/15522/commits/9f0879f8d8205d76eda2a903218182dceaee9a77.